src/cost.py

```diff
@@ -17,8 +17,8 @@ def __init__(self, data):
     def cost(self, items):
```

Comment:
You need a items parameter which describes which items you want to put in your bag.
The cost function calculates the weight of the items in your bag

Reply:
You should use a binary list or np.array as the items parameter.
exmpl:
[0, 1, 0, 1, 1, 1, 1, 0, 1, 0]